### ISSUE TITLE
tde-lib has unstated prerequisite of schema database

### ISSUE THREAD

**Issue by masyukun** [marklogic-community/data-explorer#188]
If your content database has "(none)" assigned to the schema database field, then tde-lib.xqy will throw an error when Creating a Query, in function tde-lib:has-tde() on line 20.

The code assumes that the database has a schema database. If this is not the case, then the error is returned:
No database with identifier 0

**Comment by markplotnick** [comment]
Agree. Also ran into this. If the requirement remains a better error message is absolutely needed.
